billing-service/src/main/java/com/petclinic/billing/datalayer/DataSetupService.java

```diff
@@ -20,16 +20,27 @@ public DataSetupService(BillService billService) {
 
     @Override
     public void run(String... args) throws Exception {
-//       BillRequestDTO b1 = new BillRequestDTO( "1", "general", "1", LocalDate.of(2021,9,19),59.99);
-        BillRequestDTO b1 = new BillRequestDTO( "1", "general", "1", LocalDate.of(2021,9,19),59.99);
+////       BillRequestDTO b1 = new BillRequestDTO( "1", "general", "1", LocalDate.of(2021,9,19),59.99);
+//        BillRequestDTO b1 = new BillRequestDTO( "1", "general", "1", LocalDate.of(2021,9,19),59.99);
```

Comment:
This is all a little confusing to me, could you explain why some of the lines are commented out and why some names were changed but some weren't. Or just a general idea of what happened here because I see a lot of lines commented out and even extra slashes, makes it a little hard to read.

Reply:
The commented lines is the old code before I added BillStatus to them, I forgot to remove the comment.